tests/test_wsi_registration.py

```diff
@@ -338,7 +340,7 @@ def test_register_output_with_initializer(
     pre_transform = np.array([[-1, 0, 337.8], [0, -1, 767.7], [0, 0, 1]])
 
     expected = np.array(
-        [[-0.99683, -0.00333, 338.69983], [-0.03201, -0.98420, 770.22941], [0, 0, 1]]
+        [[-0.98454, -0.00708, 397.95628], [-0.01024, -0.99752, 684.81131], [0, 0, 1]]
```

Comment:
Just wondering what happened here to need changing?

Reply:
This is because I later added a code to fix the translation offset.
https://github.com/TissueImageAnalytics/tiatoolbox/blob/e03db788820e0be752b1619791e5751b6df3a464/tiatoolbox/tools/registration/wsi_registration.py#L1073-L1079